proto/common/link.proto

```diff
@@ -5,7 +5,9 @@ package sonrio.sonr.common;
 option go_package = "github.com/sonr-io/sonr/x/common/types";
 
 enum LinkKind {
```

Comment:
What is the purpose of bucket and object here?

Reply:
these will be used for objects, and buckets to denote the link "kind" such that we can delimitate the context of the `cid`